Drivers/CAN/can.h

```diff
@@ -0,0 +1,108 @@
+/**
+ * @file can.h
+ * @author Hamza Iqbal (iqbal.ha@northeastern.edu)
+ * @brief CAN Driver
+ * @date 2023-06-14
+ * 
+ * https://www.st.com/resource/en/user_manual/um1725-description-of-stm32f4-hal-and-lowlayer-drivers-stmicroelectronics.pdf
+ * CAN Documentation can be found in pages 87-113
+ */
+
+#ifndef CAN_DRIVER_H
+#define CAN_DRIVER_H
+
+#include "stm32f4xx_hal.h"
+
+
+/*
+    Macro Definitions
+    *************************************************************
+*/
+#define BAUD_RATE           1000000     //Change if needed
```

Comment:
baud rate should probably be unused, and go into a central can config struct that we initialize per line